### PR TITLE
Added e2e Tests

### PR DESCRIPTION
**Pull request by MonikaSharma06**
To run e2e tests-
`cd src/frontend`
`ng e2e`

### CODE REVIEW

File: src/frontend/e2e/app.po.ts [hunk 1]
```diff
@@ -1,11 +1,78 @@
...
+    browser.ignoreSynchronization = true;
+    return element(by.css('#app .nav-right span')).getText();
+  }
+
+  getConfigTitles() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('#app app-configuration-manager')).getText();
+  }
+
+  getTitleSchedules() {
```
Comment (praveen-garg) [inline src/frontend/e2e/app.po.ts]: getSchedulesTitle ?

File: src/frontend/e2e/app.po.ts [hunk 2]
```diff
@@ -1,11 +1,78 @@
...
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-scheduled-process .is-parent > div > header p')).getText();
+  }
+
+  getTitleCreateSchedule() {
```
Comment (praveen-garg) [inline src/frontend/e2e/app.po.ts]: getCreateScheduleTitle?

Comment (praveen-garg) [reply src/frontend/e2e/app.po.ts]: with both of above, either we say `getTitle<Component>` or `get<Component>Title` i.e. stay consistent

File: src/frontend/e2e/app.po.ts [hunk 3]
```diff
@@ -1,11 +1,78 @@
...
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-scheduled-process app-list-tasks header > div')).getText();
+  }
+
+  getAuditLogsTitle() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-audit-log nav small')).getText();
+  }
+
+  getSelectTag() {
```
Comment (praveen-garg) [inline src/frontend/e2e/app.po.ts]: you mean getAuditLogsSelect ?

File: src/frontend/e2e/app.po.ts [hunk 4]
```diff
@@ -1,11 +1,78 @@
...
+    browser.ignoreSynchronization = true;
+    return element.all(by.css('app-audit-log div:nth-child(1) select')).count();
+  }
+
+  getInputTag() {
```
Comment (praveen-garg) [inline src/frontend/e2e/app.po.ts]: getAuditLogsInputTag

File: src/frontend/e2e/app.po.ts [hunk 5]
```diff
@@ -1,11 +1,78 @@
...
+  }
+
+  getAssetsTitle() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-assets div:nth-child(1) > div > article > h5 > small')).getText();
+  }
+
+  getAssetReadingsTitle() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('app-assets div:nth-child(2) > div > article > h5 > small')).getText();
```
Comment (praveen-garg) [inline src/frontend/e2e/app.po.ts:84]: add a comment how its diff with getAssetTitle

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by praveen-garg** [comment]
Can not we run with `yarn e2e` ?

File: src/frontend/e2e/app.po.ts (hunk 5 above)
Comment (MonikaSharma06) [reply src/frontend/e2e/app.po.ts:84]: In 'getAssetTitle', there is `div:nth-child(1)` and in 'getAssetReadingsTitle', there is `div:nth-child(2)`

**Comment by MonikaSharma06** [comment]
yes, we can run `yarn e2e` also.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**